src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="#" method="POST"
+      class="container"
+      data-qa="big">
+
+      <input type="search" class="input"
+      placeholder="Try &quot;Los Angeles&quot;">
+
+      <img src="images/Search.svg" alt="search"
+        class="image_container">
+
+    </form>
+
+    <form action="#" method="POST"
+      class="container"
+      data-qa="small">
```

Comment:
```suggestion
    <form 
      action="#" 
      method="POST"
      class="container"
      data-qa="small"
    >
```
Открывающая и закрывающая скобка тега должны быть на одной линии

Reply:
сделал